backend-api/src/functions/exampleLambda/exampleLambdaConfig.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Config,
+  getRequiredEnvironmentVariables,
+  MissingEnvVarError,
+} from "../common/utils/environment";
+import { Result } from "../utils/result";
+
+const REQUIRED_ENVIRONMENT_VARIABLES = ["SESSIONS_TABLE"] as const;
```

Comment:
What is the value in asking for the envVars you want like you do here vs having envVars defined for each lambda and just getting all of them?

Reply:
Not sure I understand the question - this would be where we define all the env vars for the lambda, it just so happens that this example lambda only has one.